setup.py

```diff
@@ -23,10 +23,11 @@ def local_scheme(version):
     #long_description="""TODO""",
     license="BSD",
     setup_requires=["setuptools_scm"],
-    install_requires=["nmigen"],
+    install_requires=["nmigen~=0.1.rc1"],
     packages=find_packages(),
     project_urls={
         "Source Code": "https://github.com/m-labs/nmigen-stdio",
         "Bug Tracker": "https://github.com/m-labs/nmigen-stdio/issues",
     },
 )
+
```

Comment:
remove

Reply:
Shouldn't there be a blank line at the end of the file?